.github/workflows/main.yml

```diff
@@ -110,3 +110,21 @@ jobs:
         with:
           name: collection
           path: theforeman-operations-*.tar.gz
+
+  molecule:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - uses: gofrolist/molecule-action@v2
+        with:
+          molecule_working_dir: roles/foreman_installer
+        env:
+          DRIVER: docker
```

Comment:
why `ENV` instead of `molecule_options: --driver docker`?

Reply:
Because:

```
ehelms@wareagle foreman_installer (add-foreman-installer)$ molecule check --driver docker
Usage: molecule check [OPTIONS]
Try 'molecule check --help' for help.

Error: no such option: --driver
ehelms@wareagle foreman_installer (add-foreman-installer)$ molecule --driver docker check
Usage: molecule [OPTIONS] COMMAND [ARGS]...
Try 'molecule --help' for help.

Error: no such option: --driver
```

Reply:
`--driver-name`

Reply:
They must have dropped that:

```
ehelms@wareagle foreman-operations-collection (add-foreman-installer)$ molecule --driver-name docker check
Usage: molecule [OPTIONS] COMMAND [ARGS]...
Try 'molecule --help' for help.

Error: no such option: --driver-name
ehelms@wareagle foreman-operations-collection (add-foreman-installer)$ molecule check --driver-name docker
Usage: molecule check [OPTIONS]
Try 'molecule check --help' for help.

Error: no such option: --driver-name
```

Reply:
Weird. Also if you pass that to`molecule test`?

Reply:
But if that's not working reliably, let's ot block.

Maybe should call the env var MOLECULE_DRIVER though

Reply:
`MOLECULE_` is a reserved namespace. So I opted for `DRIVER_NAME` for closer consistency.